elements/rh-back-to-top/demo/color-context.html

```diff
@@ -0,0 +1,69 @@
+<rh-context-demo>
+  <a id="top" href="#bottom">Go to bottom</a>
+  <p>Scroll down or press tab to see the back to top button</p>
+  <a id="bottom" href="#top">Go to top</a>
+</rh-context-demo>
+
+<rh-back-to-top href="#top">Back to top</rh-back-to-top>
+
+<script type="module">
+  import '@rhds/elements/rh-back-to-top/rh-back-to-top.js';
+  import '@patternfly/elements/pf-icon/pf-icon.js';
+  import '@rhds/elements/lib/elements/rh-context-demo/rh-context-demo.js';
+</script>
+
+<style>
+  [data-demo] {
```

Comment:
Maybe put this on the main element...

Reply:
I need `<main>` to represent the element being overflowed, and a element inside of it to overflow it to cause the scroll.  Also was wanting to avoid having to set `scrollable-selector` in every demo I have a script that handles which to target `main` or `window` given if main is scrollable or not.

What maybe better is just wrapping the demo content in a `<div id="overflow">` instead of relying on `[data-demo]` which doesn't exist in playground.  However now we are also adding more html, but should fix the playground view maybe?

I was attempting to try and add as little "extra" code to get around the inconsistencies between dev-server, demo docs/playground views.   TBH I'm kinda over futzing with it.  Feel free to modify how you see fit.  IMHO time would be better served fixing the demo experience as a whole.